fdb-relational-server/src/main/java/com/apple/foundationdb/relational/server/RelationalServer.java

```diff
@@ -131,7 +138,7 @@ RelationalServer start() throws IOException {
         // Create access to backing database.
         // TODO: Make this multi-query/-tenant/-database!
         try {
-            frl = new FRL();
+            frl = new FRL(com.apple.foundationdb.relational.api.Options.NONE, clusterFile);
```

Comment:
We also have `Options` from apache cli, so this needs to be fully qualified (or statically imported)